app/models/concerns/galtersufia/generic_file/license_data.txt

```diff
@@ -0,0 +1,77 @@
+{
+  'All Rights Reserved': {
+    "name": 'All Rights Reserved'
+  },
+  'http://creativecommons.org/publicdomain/mark/1.0/': {
```

Comment:
Just curious, how were you able to map this url to their name and licenseId? At a glance, it seems to me the map on SPDX is removing the '/legalcode' part of the "seeAlso"/Other webpages.

Reply:
It was really tedious, but I just went through the SPDX JSON files and searched for each license that was included in the Sufia initializer/configuration. The licenses were all mapped within the initializer as the name of the license to the URL fo the website of that license. So, basically I just took the URLs from the initializer as a key and then either copied the relevant JSON from the files I downloaded or just added the license id and name for the licenses that were not included on the SPDX.

Reply:
Alright, updated.